api/backup.go

```diff
@@ -18,7 +19,61 @@ func (s *Server) BackupTargetList(w http.ResponseWriter, req *http.Request) erro
 	if err != nil {
 		return errors.Wrap(err, "failed to list backup targets")
 	}
-	apiContext.Write(toBackupTargetCollection(backupTargets))
+	apiContext.Write(toBackupTargetCollection(backupTargets, apiContext))
+	return nil
+}
+
+func (s *Server) BackupTargetUpdateAll(w http.ResponseWriter, req *http.Request) error {
```

Comment:
Rename `SyncAllBackupTargets` if this is what you want to achieve.

Reply:
Used `BackupTargetSyncAll` to keep the same prefix for group methods in the `api` level.
Thanks.